ext/drutil/drutil.h

```diff
@@ -110,6 +110,24 @@ DR_EXPORT
  * to be passed in.
  * For single-instruction string loops, returns the size referenced
  * by each iteration.
+ *
+ * If the instruction is part of the xsave family of instructions, this
+ * returns an incomplete computation of the xsave instruction's written
+ * xsave area's size. Specifically, it
+ *
+ * - Ignores the user state mask components set in edx:eax, because it is
+ *   a dynamic value at runtime. The real output size of xsave depends on
+ *   the instruction's user state mask AND the user state mask as supported
+ *   by the CPU based on XCR0.
+ * - Ignores supervisor state component PT (bit 8).
```

Comment:
bit 8 of what?  Similarly for all the "bit N" refs below.

Reply:
Done/refined.